arrows/serialize/json/klv/load_save_klv.cxx

```diff
@@ -573,7 +574,7 @@ class klv_json_saver : public klv_json_base< save_archive >
 
   void save( klv_blob const& value )
```

Comment:
maybe call this save_base64 or something? "save" in combination with the blob input type seems non-descriptive.